version: '3.9'

volumes:
  postgres-data:
    external: false # Внешние волюмы не привязываются к композу и при docker-compose down --volumes не удаляются
  redis-data:
    external: false

networks:
  litminka-net: 
    external: false

services:
  postgre-db:
    container_name: litminka-db
    image: postgres:15-alpine
    environment:
      POSTGRES_DB: "litminer"
      POSTGRES_USER: "arckontyr"
      POSTGRES_PASSWORD: "arc"
    ports:
      - target: 5432
        published: 5433
    volumes:
      - postgres-data:/var/lib/postgresql/data
    networks:
      - litminka-net

  redis:
    container_name: redis
    image: redis:7-alpine
    volumes:
      - redis-data:/data
    ports:
      - target: 6379
        published: 6380
    networks:
      - litminka-net
